articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,166 @@
+
+## Availability sets
+
+An Availability Set is a logical grouping capability that you can use in Azure to ensure that the VM resources you place within it are isolated from each other when they are deployed within an Azure datacenter. Azure ensures that the VMs you place within an Availability Set run across multiple physical servers, compute racks, storage units, and network switches. If a hardware or Azure software failure occurs, only a subset of your VMs are impacted, and your overall application stays up and continues to be available to your customers. Availability Sets are an essential capability when you want to build reliable cloud solutions.
+
+Let’s consider a typical VM-based solution where you might have four front-end web servers and 2 back-end VMs hosting databases. With Azure Stack, you’d want to define two availability sets before you deploy your VMs: one availability set for the web tier and one availability set for the back tier. When you create a new VM you can then specify the availability set as a parameter to the `az vm` create command, and Azure Stack automatically ensures that the VMs you create within the available set are isolated across multiple physical hardware resources. If the physical hardware that one of your Web Server or back-end VMs is running on has a problem, you know that the other instances of your Web Server and back-end VMs remain running because they are on different hardware.
+
+## Creating an availability set via the portal
+
+1. The first step when initially creating an availability set is to create the resource group for the availability set.
+
+    ![Microsoft Azure availability set step 1](images/azs-portal-create-availability-set1.png)
+
+2. Name the resource group.
+
+    ![Microsoft Azure availability set step 2](images/azs-portal-create-availability-set2.png)
+
+3. Once the resource group has been created if you navigate into it and select **Overview** then you will be able to select **Create resources**.
+
+    ![Microsoft Azure availability set step 3](images/azs-portal-create-availability-set3.png)
+
+4. This will load the marketplace, if you search for **Availability Set**, you will be given the option to create one.
+
+    ![Microsoft Azure availability set step 4](images/azs-portal-create-availability-set4.png)
+
+5. This will bring up the **Create availability** set blade. Here you can set the name of the set and how many fault and update domains you want it to have. The resource group should be the same as the one you created earlier. As for the option for managed disks, select the option for which matches the VMs disks you will be adding to the set. Once done select **Create** and the availability set will be deployed.
```

Comment:
In the **Create availability set** blade, you can set the name of the set and how many fault and update domains you want it to have. For the **Resource group**, select the resource group you created earlier and for **Use managed disks**, select the option that matches the VM disks you'll be adding to the set. When you're done, click **Create** to deploy the availability set.

Reply:
done